pkg/cli/schemaherokubectlcli/fixtures.go

```diff
@@ -33,7 +35,10 @@ func FixturesCmd() *cobra.Command {
 	cmd.Flags().String("input-dir", "", "directory to read schema files from")
 	cmd.Flags().String("output-dir", "", "directory to write fixture files to")
 
-	cmd.MarkFlagRequired("driver")
+	err := cmd.MarkFlagRequired("driver")
+	if err != nil {
+		fmt.Print("error ", err)
```

Comment:
Same here. Printing the error and continuing might not be sufficient. If there's an error, this function doesn't return one. Can the caller handle returning an error? Alternatively, we can panic here because this is in the init function.